### ISSUE TITLE
tokenizePaymentFormData() does not return cardHolder/cardType/issuer fields in response

### ISSUE THREAD

**Issue by oleksandr-tkach** [hipay/hipay-fullservice-sdk-js#34]
The problem is that in _processObjectPayload method when initialising var mapping = instance._mapping[key];, mapping for some keys is undefined because key in some cases is in snake case notation (f.e. "card_holder") but the _mapping actual keys are in camel case notation (f.e. "cardHolder").
So at the end I get a response object with some basic fields like token, brand, country, pan (obviously because for one word snake case and camel case values are same).

so basically to fix this you just need to fix naming or add a quick inside _processObjectPayload method:
`
...
if (typeof instance._mapping === 'object') {
   var camelCasedKey = key.replace(/_([a-z])/g, function (g) {
      return g[1].toUpperCase();
   });
   var mapping = instance._mapping[camelCasedKey];
...
}
`

**Comment by legithubdeaymeric** [comment]
Thanks for your feedback.

We correct that quickly.

**Comment by oleksandr-tkach** [comment]
Thank you, sir.

**Comment by FlorianBurgevin** [comment]
Hello,
The problem is fixed in v2.0.2 ( https://github.com/hipay/hipay-fullservice-sdk-js/commit/b677d2afbe8d6ac4c1ad2995e33c40e94ea4b099 )

Can you test and tell me if I can close the issue?

**Comment by oleksandr-tkach** [comment]
@FlorianBurgevin yes, it works as expected after your commit. Thanks.
